### ISSUE TITLE
Better UX for credentials expired

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/sdr-client#77]
```
$ ./exe/sdr --service-url https://sdr-api-stage.stanford.edu deposit --admin-policy druid:cx087xf3798 --source_id justin:test1  --collection druid:bm578rj4198
I, [2020-02-27T14:29:39.323711 #49993]  INFO -- : checking to see if files exist
I, [2020-02-27T14:29:39.392618 #49993]  INFO -- : Starting upload metadata: {:access=>{}, :type=>"http://cocina.sul.stanford.edu/models/book.jsonld", :administrative=>{:hasAdminPolicy=>"druid:cx087xf3798"}, :identification=>{:sourceId=>"justin:test1"}, :structural=>{:isMemberOf=>"druid:bm578rj4198"}}
Traceback (most recent call last):
	5: from ./exe/sdr:112:in `<main>'
	4: from /Users/jcoyne85/workspace/sul-dlss/sdr-client/lib/sdr_client/cli.rb:9:in `start'
	3: from /Users/jcoyne85/workspace/sul-dlss/sdr-client/lib/sdr_client/deposit.rb:36:in `run'
	2: from /Users/jcoyne85/workspace/sul-dlss/sdr-client/lib/sdr_client/deposit/process.rb:38:in `run'
	1: from /Users/jcoyne85/workspace/sul-dlss/sdr-client/lib/sdr_client/deposit/process.rb:57:in `upload_metadata'
/Users/jcoyne85/workspace/sul-dlss/sdr-client/lib/sdr_client/deposit/process.rb:66:in `unexpected_response': There was an error with your credentials. Perhaps they have expired? (RuntimeError)
```

**Comment by mjgiarlo** [comment]
Still needs to be done? Helpful for H2 work?

**Comment by mjgiarlo** [comment]
Fixed by #67
